src/Module/Photo.php

```diff
@@ -201,7 +201,7 @@ private static function getAvatar($uid, $type="avatar", $customsize)
 				} else {
 					$url = Contact::getDefaultAvatar($contact, Proxy::SIZE_SMALL);
 				}
-				return MPhoto::createPhotoForExternalResource($url, local_user());
+				return MPhoto::createPhotoForExternalResource($url, 0);
```

Comment:
Please create a default value for this parameter with an explanation of its meaning (public image?)

Reply:
Done